src/Parsing/ParserState.php

```diff
@@ -404,11 +404,11 @@ public function streql($string1, $string2, $caseInsensitive = true): bool
     }
 
     /**
-     * @param int $iAmount
+     * @param int $amount
```

Comment:
We could possibly improve this parameter name while renaming it.  Perhaps `$characterCount` or `$numberOfCharacters`.